requirements/make_base.py

```diff
@@ -27,7 +28,10 @@
 
 def as_nightly(name: str) -> str:
     if name == "scipp":
-        return "https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"  # noqa: E501
+        version = f"py{sys.version_info.major}{sys.version_info.minor}"
```

Comment:
Shouldn't this be 
```suggestion
        version = f"cp{sys.version_info.major}{sys.version_info.minor}"
```
?

Reply:
Yes, fixed.